packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/color.ts

```diff
@@ -93,6 +93,8 @@ function spyToRegex(spyName: string) {
       return /^(?<packageName>(.*\/)*)(?<filename>.*\.rb+)(?<line_info>.*)$/;
     case 'nodespy':
       return /^(\.\/node_modules\/)?(?<packageName>[^/]*)(?<filename>.*\.?(jsx?|tsx?)?):(?<functionName>.*):(?<line_info>.*)$/;
+    case 'javaspy':
```

Comment:
@petethepig is this correct (and does the client send this already or do we need to make a change somewhere)